app/policies/school_policy.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class SchoolPolicy
+  class Scope
+    attr_reader :user, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      scope.order(:name)
```

Comment:
I don't think we should order in here as this should only be concerned with authorisation. We should order in the controller (or maybe a scope on the model 🤷 )